source/api_documentation/network_integration/call_recording_condition/index.rst

```diff
@@ -0,0 +1,184 @@
+      "root": {
+        "node_type": "Connect",
+        "destination_phone_number": "8056173768",
+        "destination_country_code": ""
+      }
+    }
+  }
+
+Record All Calls
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To record all calls (and to remove an existing call recording condition), set ``"record_calls"`` to ``true`` and set ``"call_recording_condition"`` to empty string ``""``.
+
+POST
+
+``https://invoca.net/api/@@CAMPAIGN_FEATURES_API_VERSION/<network_id>/advertisers/<advertiser_id_from_network>/advertiser_campaigns/<advertiser_campaign_id_from_network>.json``
+
+Example Request Body
+
+.. code-block:: json
+
+  {
+    "ivr_tree": {
+      "record_calls": true,
+      "call_recording_condition": "",
+      "root": {
+        "node_type": "Connect",
+        "destination_phone_number": "8056173768",
+        "destination_country_code": ""
+      }
+    }
+  }
+
+Record No Calls
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To record no calls (and to remove an existing call recording condition), set ``"record_calls"`` to ``false`` and set ``"call_recording_condition"`` to empty string ``""``.
+
+(If ``"record_calls"`` is ``false``, no calls will be recorded even if there is a ``"call_recording_condition"`` set.)
+
+POST
+
+``https://invoca.net/api/@@CAMPAIGN_FEATURES_API_VERSION/<network_id>/advertisers/<advertiser_id_from_network>/advertiser_campaigns/<advertiser_campaign_id_from_network>.json``
+
+Example Request Body
+
+.. code-block:: json
+
+  {
+    "ivr_tree": {
+      "record_calls": false,
+      "call_recording_condition": "",
+      "root": {
+        "node_type": "Connect",
+        "destination_phone_number": "8056173768",
+        "destination_country_code": ""
+      }
+    }
+  }
+
+Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+  :widths: 30 50 30
+  :header-rows: 1
+  :class: parameters
+
+  * - Condition
+    - Details
+    - Examples
+
+  * - during_hours
+    - True if the caller is calling during the hours specified in the campaign.
+    - during_hours
+
+  * - repeat
+    - True if the caller has already called this campaign in the last N days (the interval N can be set on the campaign; the default is 30 days).
+    - repeat
+
+  * - landline
+    - True if the caller is calling from a landline phone.
+    - landline
+
+  * - mobile
+    - True if the caller is calling from a mobile phone.
+    - mobile
+
+  * - pressed[key]
+    - True if the caller pressed the key. Supported characters (case sensitive): [1-9, a-z, W, N]. For a sub-menu keypress, join characters in their expected order with spaces (e.g. 1 1 2).
+    - pressed[3]
+
+  * - custom_data[field_partner_name]
+    - * True if the custom data field value matches (=) the expected value
+      * True if the custom data field value does not match (!=) the expected value
+      * True if the custom data field value contains (contains) the expected value
+      * True if the custom data field value is present (present)
```

Comment:
Should "custom data field" be "marketing data field"?

Reply:
As discussed in slack, going to leave it as since all of the api docs currently refer to "custom data"